src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\hypertitle{Stochastic processes}{6}{INMA}{1731}{2019}{June}{Majeure}
+{Alice Borbáth \and Olivier Leblanc \and Julien Verecken}
+{Pierre-Antoine Absil and Luc Vandendorpe}
+
+Exam during 3h with 3 questions: 2 questions by PAA et 1 question by LV.
+
+\section{PAA 1}
+
+Let $G(z)$ be an LTI system which is stable and causal. 
+One inputs a white signal $X$ of variance $\sigma_x^2$ in the system and outputs $Y$.
+The transfer function is given by :
+\begin{equation}
+    G(z)=\frac{z}{\left(z-\frac{1-\epsilon}{\sqrt{2}}(1+j)\right)\left(z-\frac{1-\epsilon}{\sqrt{2}}(1-j)\right)}
+\end{equation}
+\indent where $0<\epsilon\ll1$. Let the input X be white noise with variance $\sigma_X^2$.
```

Comment:
```suggestion
where $0<\epsilon\ll1$.
```

This is already written 3 lines higher (and with a different variance, so I picked the one I thought looked the most logical and in tune with the rest of the notations). The indent just looks ugly, but you can keep it if you wish.